modules/cli/src/main/scala/scala/cli/commands/Doctor.scala

```diff
@@ -79,7 +79,10 @@ object Doctor extends ScalaCommand[DoctorOptions] {
       .split(pathSeparator)
       .map(d => if (d == ".") pwd else d) // on unix a bare "." counts as the current dir
       .map(_ + "/scala-cli")
-      .filter(f => os.isFile(os.Path(f)))
+      .filter { f =>
+        val p: os.Path = if f.startsWith("/") then os.Path(f) else os.RelPath(f).resolveFrom(os.pwd)
+        os.isFile(p)
+      }
```

Comment:
Just passing `os.pwd` to `os.Path` should fix this:
```suggestion
      .filter(f => os.isFile(os.Path(f, os.pwd)))
```